packages/apps/human-app/frontend/src/modules/signup/worker/index.ts

```diff
@@ -0,0 +1 @@
+export * from './views/sign-up-worker.page';
```

Comment:
For further refactoring would be nice to export from `index.ts` only the things that should be accessible outside, not everything

Reply:
@mpblocky